aom/src/main/java/com/nedap/archie/rminfo/MetaModel.java

```diff
@@ -81,121 +114,107 @@ public ObjectMapper getJsonObjectMapper() {
     @Override
     public boolean isMultiple(String typeName, String attributeNameOrPath) {
         MultiplicityInterval multiplicityInterval = referenceModelPropMultiplicity(typeName, attributeNameOrPath);
-        if(multiplicityInterval == null) {
+        if (multiplicityInterval == null) {
             return false;// by default, false if unknown property
         }
         return multiplicityInterval.isUpperUnbounded() || multiplicityInterval.getUpper() > 1;
     }
 
     private boolean isMultiple(BmmProperty<?> bmmProperty) {
-        if(bmmProperty == null) {
+        if (bmmProperty == null) {
             return false;
-        } else if(bmmProperty instanceof BmmContainerProperty) {
-            return bmmProperty != null && ((BmmContainerProperty) bmmProperty).getCardinality().has(2);
+        } else if (bmmProperty instanceof BmmContainerProperty) {
+            return ((BmmContainerProperty) bmmProperty).getCardinality().has(2);
```

Comment:
bmmProperty is always non-null at this point, so the `bmmProperty != null` check is removed.